Misc/NEWS.d/next/Tests/2025-06-11-16-52-49.gh-issue-135401.ccMXmL.rst

```diff
@@ -0,0 +1 @@
+Add a new GitHub CI job to test python's ssl module with AWS-LC as the backing cryptography and TLS library.
```

Comment:
Is there a better link for AWS-LC? Ideally we wouldn't directly link to the repo, but to the documentation or other landing page. I can't easily find such a link, though.
```suggestion
Add a new GitHub CI job to test the :py:mod:`ssl` module with AWS-LC__ as the backing cryptography and TLS library.

__ https://github.com/aws/aws-lc
```

Reply:
This is probably the best documentation https://github.com/aws/aws-lc/blob/main/README.md

I'll include this update in my next commit unless you beat me to it.

Reply:
Yes, that's the best and most consistently up-to-date documentation entry point.